src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/TopNav.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.page.IssueCreateDialog
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class TopNav(val driver: WebDriver) {
+    internal fun openIssueCreateDialog(): IssueCreateDialog {
+        createButton()!!.click()
```

Comment:
So if we can't access `create` button, the action will fail. Don't we want to wait for it to be present?

Reply:
Yeah, I wanted it to fail. But there's no harm in waiting for the link.